server/src/game/role_list.rs

```diff
@@ -324,14 +482,14 @@ impl RoleSet{
                 vec![
                     Role::Godfather, Role::Counterfeiter,
                     Role::Impostor, Role::Recruiter,
-                    Role::Mafioso
+                    Role::Mafioso, Role::MafiaKillingWildcard

```

Comment:
Should probably ask Sammy about this - not sure if he omitted these on purpose or not